### ISSUE TITLE
apt-get failed this repos

### ISSUE THREAD

**Issue by ytooyama** [NVIDIA/nvidia-docker#642]
### 1. Issue or feature description

NVIDIA Docker 2 Install Failed on Ubuntu Server 16.04.3.

### 2. Steps to reproduce the issue

When I executed this command.

```
$ curl -s -L https://nvidia.github.io/nvidia-docker/gpgkey | sudo apt-key add -
$ curl -s -L https://nvidia.github.io/nvidia-docker/ubuntu16.04/amd64/nvidia-docker.list | \
  sudo tee /etc/apt/sources.list.d/nvidia-docker.list
$ sudo apt-get update
```

Result:
Only github.io repo are HTTP 403.

How to Fix this error?

```
ubuntu@gpu:~$ sudo apt-get update
Ign:1 https://nvidia.github.io/libnvidia-container/ubuntu16.04/amd64  InRelease
Ign:2 https://nvidia.github.io/nvidia-container-runtime/ubuntu16.04/amd64  InRelease
Ign:3 https://nvidia.github.io/nvidia-docker/ubuntu16.04/amd64  InRelease
Ign:4 https://nvidia.github.io/libnvidia-container/ubuntu16.04/amd64  Release
Ign:5 https://nvidia.github.io/nvidia-container-runtime/ubuntu16.04/amd64  Release
Ign:6 https://nvidia.github.io/nvidia-docker/ubuntu16.04/amd64  Release
Ign:7 https://nvidia.github.io/libnvidia-container/ubuntu16.04/amd64  Packages
Ign:8 https://nvidia.github.io/libnvidia-container/ubuntu16.04/amd64  Translation-en_US
Ign:9 https://nvidia.github.io/libnvidia-container/ubuntu16.04/amd64  Translation-en
Ign:10 https://nvidia.github.io/nvidia-container-runtime/ubuntu16.04/amd64  Packages
Ign:11 https://nvidia.github.io/nvidia-container-runtime/ubuntu16.04/amd64  Translation-en_US
Ign:12 https://nvidia.github.io/nvidia-container-runtime/ubuntu16.04/amd64  Translation-en
Ign:13 https://nvidia.github.io/nvidia-docker/ubuntu16.04/amd64  Packages
Ign:14 https://nvidia.github.io/nvidia-docker/ubuntu16.04/amd64  Translation-en_US
Ign:15 https://nvidia.github.io/nvidia-docker/ubuntu16.04/amd64  Translation-en
Ign:7 https://nvidia.github.io/libnvidia-container/ubuntu16.04/amd64  Packages
Ign:8 https://nvidia.github.io/libnvidia-container/ubuntu16.04/amd64  Translation-en_US
Ign:9 https://nvidia.github.io/libnvidia-container/ubuntu16.04/amd64  Translation-en
Ign:10 https://nvidia.github.io/nvidia-container-runtime/ubuntu16.04/amd64  Packages
Ign:11 https://nvidia.github.io/nvidia-container-runtime/ubuntu16.04/amd64  Translation-en_US
Ign:12 https://nvidia.github.io/nvidia-container-runtime/ubuntu16.04/amd64  Translation-en
Ign:13 https://nvidia.github.io/nvidia-docker/ubuntu16.04/amd64  Packages
Ign:14 https://nvidia.github.io/nvidia-docker/ubuntu16.04/amd64  Translation-en_US
Ign:15 https://nvidia.github.io/nvidia-docker/ubuntu16.04/amd64  Translation-en
Ign:7 https://nvidia.github.io/libnvidia-container/ubuntu16.04/amd64  Packages
Ign:8 https://nvidia.github.io/libnvidia-container/ubuntu16.04/amd64  Translation-en_US
Ign:9 https://nvidia.github.io/libnvidia-container/ubuntu16.04/amd64  Translation-en
Ign:10 https://nvidia.github.io/nvidia-container-runtime/ubuntu16.04/amd64  Packages
Ign:11 https://nvidia.github.io/nvidia-container-runtime/ubuntu16.04/amd64  Translation-en_US
Ign:12 https://nvidia.github.io/nvidia-container-runtime/ubuntu16.04/amd64  Translation-en
Ign:13 https://nvidia.github.io/nvidia-docker/ubuntu16.04/amd64  Packages
Ign:14 https://nvidia.github.io/nvidia-docker/ubuntu16.04/amd64  Translation-en_US
Ign:15 https://nvidia.github.io/nvidia-docker/ubuntu16.04/amd64  Translation-en
Ign:7 https://nvidia.github.io/libnvidia-container/ubuntu16.04/amd64  Packages
Ign:8 https://nvidia.github.io/libnvidia-container/ubuntu16.04/amd64  Translation-en_US
Ign:9 https://nvidia.github.io/libnvidia-container/ubuntu16.04/amd64  Translation-en
Ign:10 https://nvidia.github.io/nvidia-container-runtime/ubuntu16.04/amd64  Packages
Ign:11 https://nvidia.github.io/nvidia-container-runtime/ubuntu16.04/amd64  Translation-en_US
Ign:12 https://nvidia.github.io/nvidia-container-runtime/ubuntu16.04/amd64  Translation-en
Ign:13 https://nvidia.github.io/nvidia-docker/ubuntu16.04/amd64  Packages
Ign:14 https://nvidia.github.io/nvidia-docker/ubuntu16.04/amd64  Translation-en_US
Ign:15 https://nvidia.github.io/nvidia-docker/ubuntu16.04/amd64  Translation-en
Ign:7 https://nvidia.github.io/libnvidia-container/ubuntu16.04/amd64  Packages
Ign:8 https://nvidia.github.io/libnvidia-container/ubuntu16.04/amd64  Translation-en_US
Ign:9 https://nvidia.github.io/libnvidia-container/ubuntu16.04/amd64  Translation-en
Ign:10 https://nvidia.github.io/nvidia-container-runtime/ubuntu16.04/amd64  Packages
Ign:11 https://nvidia.github.io/nvidia-container-runtime/ubuntu16.04/amd64  Translation-en_US
Ign:12 https://nvidia.github.io/nvidia-container-runtime/ubuntu16.04/amd64  Translation-en
Ign:13 https://nvidia.github.io/nvidia-docker/ubuntu16.04/amd64  Packages
Ign:14 https://nvidia.github.io/nvidia-docker/ubuntu16.04/amd64  Translation-en_US
Ign:15 https://nvidia.github.io/nvidia-docker/ubuntu16.04/amd64  Translation-en
Err:7 https://nvidia.github.io/libnvidia-container/ubuntu16.04/amd64  Packages
  Received HTTP code 403 from proxy after CONNECT
Ign:8 https://nvidia.github.io/libnvidia-container/ubuntu16.04/amd64  Translation-en_US
Ign:9 https://nvidia.github.io/libnvidia-container/ubuntu16.04/amd64  Translation-en
Err:10 https://nvidia.github.io/nvidia-container-runtime/ubuntu16.04/amd64  Packages
  Received HTTP code 403 from proxy after CONNECT
Ign:11 https://nvidia.github.io/nvidia-container-runtime/ubuntu16.04/amd64  Translation-en_US
Ign:12 https://nvidia.github.io/nvidia-container-runtime/ubuntu16.04/amd64  Translation-en
Err:13 https://nvidia.github.io/nvidia-docker/ubuntu16.04/amd64  Packages
  Received HTTP code 403 from proxy after CONNECT
Ign:14 https://nvidia.github.io/nvidia-docker/ubuntu16.04/amd64  Translation-en_US
Ign:15 https://nvidia.github.io/nvidia-docker/ubuntu16.04/amd64  Translation-en
Hit:16 http://archive.ubuntu.com/ubuntu xenial InRelease
Hit:17 http://archive.ubuntu.com/ubuntu xenial-updates InRelease
Hit:18 http://archive.ubuntu.com/ubuntu xenial-backports InRelease
Hit:19 http://archive.ubuntu.com/ubuntu xenial-security InRelease
Reading package lists... Done
W: The repository 'https://nvidia.github.io/libnvidia-container/ubuntu16.04/amd64  Release' does not have a Release file.
N: Data from such a repository can't be authenticated and is therefore potentially dangerous to use.
N: See apt-secure(8) manpage for repository creation and user configuration details.
W: The repository 'https://nvidia.github.io/nvidia-container-runtime/ubuntu16.04/amd64  Release' does not have a Release file.
N: Data from such a repository can't be authenticated and is therefore potentially dangerous to use.
N: See apt-secure(8) manpage for repository creation and user configuration details.
W: The repository 'https://nvidia.github.io/nvidia-docker/ubuntu16.04/amd64  Release' does not have a Release file.
N: Data from such a repository can't be authenticated and is therefore potentially dangerous to use.
N: See apt-secure(8) manpage for repository creation and user configuration details.
E: Failed to fetch https://nvidia.github.io/libnvidia-container/ubuntu16.04/amd64/Packages  Received HTTP code 403 from proxy after CONNECT
E: Failed to fetch https://nvidia.github.io/nvidia-container-runtime/ubuntu16.04/amd64/Packages  Received HTTP code 403 from proxy after CONNECT
E: Failed to fetch https://nvidia.github.io/nvidia-docker/ubuntu16.04/amd64/Packages  Received HTTP code 403 from proxy after CONNECT
E: Some index files failed to download. They have been ignored, or old ones used instead.
ubuntu@gpu:~$ sudo apt-get install nvidia-docker2=2.0.2+docker1.13.1-1 nvidia-container-runtime=1.1.1+docker1.13.1-1
Reading package lists... Done
Building dependency tree       
Reading state information... Done
E: Unable to locate package nvidia-docker2
E: Unable to locate package nvidia-container-runtime
```

### 3. Information to [attach](https://help.github.com/articles/file-attachments-on-issues-and-pull-requests/) (optional if deemed irrelevant)

 - [x] Kernel version from `uname -a`

```
$ uname -a
Linux gpu2 4.4.0-112-generic #135-Ubuntu SMP Fri Jan 19 11:48:36 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux

$ cat /proc/version_signature
Ubuntu 4.4.0-112.135-generic 4.4.98
```
 - [ ] Any relevant kernel output lines from `dmesg`
 - [x] Driver information from `nvidia-smi -a`

```
ubuntu@gpu2:/etc/apt/sources.list.d$ nvidia-smi -a

==============NVSMI LOG==============

Timestamp                           : Tue Feb 27 05:00:53 2018
Driver Version                      : 390.30

Attached GPUs                       : 1
GPU 00000000:01:00.0
    Product Name                    : GeForce GTX 1080
    Product Brand                   : GeForce
    Display Mode                    : Enabled
    Display Active                  : Disabled
    Persistence Mode                : Disabled
    Accounting Mode                 : Disabled
    Accounting Mode Buffer Size     : 1920
    Driver Model
        Current                     : N/A
        Pending                     : N/A
    Serial Number                   : 0322016055278
    GPU UUID                        : GPU-61e6272e-43f6-63b0-6429-9d3f0092caac
    Minor Number                    : 0
    VBIOS Version                   : 86.04.17.00.80
    MultiGPU Board                  : No
    Board ID                        : 0x100
    GPU Part Number                 : N/A
    Inforom Version
        Image Version               : G001.0000.01.03
        OEM Object                  : 1.1
        ECC Object                  : N/A
        Power Management Object     : N/A
    GPU Operation Mode
        Current                     : N/A
        Pending                     : N/A
    GPU Virtualization Mode
        Virtualization mode         : None
    PCI
        Bus                         : 0x01
        Device                      : 0x00
        Domain                      : 0x0000
        Device Id                   : 0x1B8010DE
        Bus Id                      : 00000000:01:00.0
        Sub System Id               : 0x61803842
        GPU Link Info
            PCIe Generation
                Max                 : 3
                Current             : 1
            Link Width
                Max                 : 16x
                Current             : 16x
        Bridge Chip
            Type                    : N/A
            Firmware                : N/A
        Replays since reset         : 0
        Tx Throughput               : 0 KB/s
        Rx Throughput               : 0 KB/s
    Fan Speed                       : 27 %
    Performance State               : P8
    Clocks Throttle Reasons
        Idle                        : Active
        Applications Clocks Setting : Not Active
        SW Power Cap                : Not Active
        HW Slowdown                 : Not Active
            HW Thermal Slowdown     : Not Active
            HW Power Brake Slowdown : Not Active
        Sync Boost                  : Not Active
        SW Thermal Slowdown         : Not Active
        Display Clock Setting       : Not Active
    FB Memory Usage
        Total                       : 8119 MiB
        Used                        : 0 MiB
        Free                        : 8119 MiB
    BAR1 Memory Usage
        Total                       : 256 MiB
        Used                        : 2 MiB
        Free                        : 254 MiB
    Compute Mode                    : Default
    Utilization
        Gpu                         : 0 %
        Memory                      : 0 %
        Encoder                     : 0 %
        Decoder                     : 0 %
    Encoder Stats
        Active Sessions             : 0
        Average FPS                 : 0
        Average Latency             : 0
    Ecc Mode
        Current                     : N/A
        Pending                     : N/A
    ECC Errors
        Volatile
            Single Bit
                Device Memory       : N/A
                Register File       : N/A
                L1 Cache            : N/A
                L2 Cache            : N/A
                Texture Memory      : N/A
                Texture Shared      : N/A
                CBU                 : N/A
                Total               : N/A
            Double Bit
                Device Memory       : N/A
                Register File       : N/A
                L1 Cache            : N/A
                L2 Cache            : N/A
                Texture Memory      : N/A
                Texture Shared      : N/A
                CBU                 : N/A
                Total               : N/A
        Aggregate
            Single Bit
                Device Memory       : N/A
                Register File       : N/A
                L1 Cache            : N/A
                L2 Cache            : N/A
                Texture Memory      : N/A
                Texture Shared      : N/A
                CBU                 : N/A
                Total               : N/A
            Double Bit
                Device Memory       : N/A
                Register File       : N/A
                L1 Cache            : N/A
                L2 Cache            : N/A
                Texture Memory      : N/A
                Texture Shared      : N/A
                CBU                 : N/A
                Total               : N/A
    Retired Pages
        Single Bit ECC              : N/A
        Double Bit ECC              : N/A
        Pending                     : N/A
    Temperature
        GPU Current Temp            : 29 C
        GPU Shutdown Temp           : 99 C
        GPU Slowdown Temp           : 96 C
        GPU Max Operating Temp      : N/A
        Memory Current Temp         : N/A
        Memory Max Operating Temp   : N/A
    Power Readings
        Power Management            : Supported
        Power Draw                  : 11.11 W
        Power Limit                 : 180.00 W
        Default Power Limit         : 180.00 W
        Enforced Power Limit        : 180.00 W
        Min Power Limit             : 90.00 W
        Max Power Limit             : 217.00 W
    Clocks
        Graphics                    : 139 MHz
        SM                          : 139 MHz
        Memory                      : 405 MHz
        Video                       : 544 MHz
    Applications Clocks
        Graphics                    : N/A
        Memory                      : N/A
    Default Applications Clocks
        Graphics                    : N/A
        Memory                      : N/A
    Max Clocks
        Graphics                    : 1911 MHz
        SM                          : 1911 MHz
        Memory                      : 5005 MHz
        Video                       : 1708 MHz
    Max Customer Boost Clocks
        Graphics                    : N/A
    Clock Policy
        Auto Boost                  : N/A
        Auto Boost Default          : N/A
    Processes                       : None
```
 - [x] Docker version from `docker version`

```
Client:
 Version:      1.13.1
 API version:  1.26
 Go version:   go1.6.2
 Git commit:   092cba3
 Built:        Thu Nov  2 20:40:23 2017
 OS/Arch:      linux/amd64

Server:
 Version:      1.13.1
 API version:  1.26 (minimum version 1.12)
 Go version:   go1.6.2
 Git commit:   092cba3
 Built:        Thu Nov  2 20:40:23 2017
 OS/Arch:      linux/amd64
 Experimental: false
```
 - [x] NVIDIA packages version from `dpkg -l '*nvidia*'` _or_ `rpm -qa '*nvidia*'`

Did not Installing. because apt-get update is failed.

 - [x] NVIDIA container library version from `nvidia-container-cli -V`

Did not Installing. because apt-get update is failed.

 - [ ] NVIDIA container library logs (see [troubleshooting](https://github.com/NVIDIA/nvidia-docker/wiki/Troubleshooting))
 - [ ] Docker command, image and tag used

**Comment by 3XX0** [comment]
Did you fix the issue, this is most likely an issue with your proxy

**Comment by ytooyama** [comment]
Thanks. I'm Try The Clean Install ageain.

**Comment by maingoh** [comment]
I also don't manage to complete the `apt-get update` on google cloud after adding nvidia docker repositories : 
```
Hit:1 http://europe-west1.gce.archive.ubuntu.com/ubuntu xenial InRelease
Hit:2 http://europe-west1.gce.archive.ubuntu.com/ubuntu xenial-updates InRelease                                                                                      
Hit:3 http://europe-west1.gce.archive.ubuntu.com/ubuntu xenial-backports InRelease                                                                                    
Hit:4 http://ppa.launchpad.net/cloud-images/gke/ubuntu xenial InRelease                                                                                               
Ign:5 http://developer.download.nvidia.com/compute/cuda/repos/ubuntu1604/x86_64  InRelease                                                             
Hit:6 http://developer.download.nvidia.com/compute/cuda/repos/ubuntu1604/x86_64  Release              
Hit:8 http://security.ubuntu.com/ubuntu xenial-security InRelease                
0% [Working]
```

It gets stuck indefinitely. `apt-get clean` didn't change anything. Only removing `/etc/apt/sources.list.d/nvidia-docker.list` unblock the `apt-get update`, but I can't install nvidia-docker :(

**Comment by maingoh** [comment]
We managed to fix my issue by installing the package `apt-transport-https`:
`sudo apt-get install apt-transport-https`
Maybe it worth adding it to the README ?

**Comment by ytooyama** [comment]
This Issue fix the upgrade apt & apt tools.
Thanks.

```
root@gpu2:~# apt list --upgradable
Listing... Done
apt/xenial-updates 1.2.26 amd64 [upgradable from: 1.2.25]
apt-transport-https/xenial-updates 1.2.26 amd64 [upgradable from: 1.2.25]
apt-utils/xenial-updates 1.2.26 amd64 [upgradable from: 1.2.25]
libapt-inst2.0/xenial-updates 1.2.26 amd64 [upgradable from: 1.2.25]
libapt-pkg5.0/xenial-updates 1.2.26 amd64 [upgradable from: 1.2.25]

root@gpu2:~# sudo apt-get update
Ign:1 http://developer.download.nvidia.com/compute/cuda/repos/ubuntu1604/x86_64  InRelease
Hit:2 http://developer.download.nvidia.com/compute/cuda/repos/ubuntu1604/x86_64  Release
Get:4 https://nvidia.github.io/libnvidia-container/ubuntu16.04/amd64  InRelease [1,106 B]
Get:5 https://nvidia.github.io/nvidia-container-runtime/ubuntu16.04/amd64  InRelease [1,103 B]
Get:6 https://nvidia.github.io/nvidia-docker/ubuntu16.04/amd64  InRelease [1,096 B]
Hit:7 http://ja.archive.ubuntu.com/ubuntu xenial InRelease
Get:8 https://nvidia.github.io/libnvidia-container/ubuntu16.04/amd64  Packages [3,504 B]
Hit:9 http://ja.archive.ubuntu.com/ubuntu xenial-updates InRelease
Get:10 https://nvidia.github.io/nvidia-container-runtime/ubuntu16.04/amd64  Packages [5,312 B]
Hit:11 http://ja.archive.ubuntu.com/ubuntu xenial-backports InRelease
Get:12 https://nvidia.github.io/nvidia-docker/ubuntu16.04/amd64  Packages [5,524 B]
Hit:13 http://ja.archive.ubuntu.com/ubuntu xenial-security InRelease
Fetched 17.6 kB in 1s (13.6 kB/s)
Reading package lists... Done
```

**Comment by ytooyama** [comment]
Thanks. Close the Issue.

**Comment by ericspod** [comment]
This issue is closed but I thought it would be worth noting that I solved this problem by replacing https with http in /etc/apt/sources.list.d/nvidia-docker.list then running apt-get update. I encountered this problem on Ubuntu 14.04 and on a Ubuntu 16.04 VM where neither apt-get update nor installing apt-transport-https worked.

**Comment by anshumang** [comment]
Neither https to http (@ericspod) nor `sudo apt-get install apt-transport-https` (@maingoh) fixes the problem for me.

My `/etc/apt/sources.list.d/nvidia-docker.list` has the following-

> deb https://nvidia.github.io/libnvidia-container/ubuntu18.04/$(ARCH) /
> deb https://nvidia.github.io/nvidia-container-runtime/ubuntu18.04/$(ARCH) /
> deb https://nvidia.github.io/nvidia-docker/ubuntu18.04/$(ARCH) /

`sudo apt-get update` produces-

> Ign:3 https://nvidia.github.io/libnvidia-container/ubuntu18.04/amd64  InRelease                                                                                                                             
> Ign:4 https://nvidia.github.io/nvidia-container-runtime/ubuntu18.04/amd64  InRelease                                                                                                                        
> Ign:5 https://nvidia.github.io/nvidia-docker/ubuntu18.04/amd64  InRelease                                                                                 
> Hit:6 http://security.ubuntu.com/ubuntu bionic-security InRelease                                                                          
> Hit:7 https://download.docker.com/linux/ubuntu bionic InRelease                                                                                                        
> Err:9 https://nvidia.github.io/libnvidia-container/ubuntu18.04/amd64  Release                                                                                          
>   Certificate verification failed: The certificate is NOT trusted. The certificate issuer is unknown.  Could not handshake: Error in the certificate verification. [IP: 185.199.110.153 443]
> Err:10 https://nvidia.github.io/nvidia-container-runtime/ubuntu18.04/amd64  Release                                  
>   Certificate verification failed: The certificate is NOT trusted. The certificate issuer is unknown.  Could not handshake: Error in the certificate verification. [IP: 185.199.110.153 443]
> Err:11 https://nvidia.github.io/nvidia-docker/ubuntu18.04/amd64  Release          
>   Certificate verification failed: The certificate is NOT trusted. The certificate issuer is unknown.  Could not handshake: Error in the certificate verification. [IP: 185.199.110.153 443]
> Hit:12 http://us.archive.ubuntu.com/ubuntu bionic InRelease                       
> Hit:13 http://us.archive.ubuntu.com/ubuntu bionic-updates InRelease
> Hit:14 http://us.archive.ubuntu.com/ubuntu bionic-backports InRelease
> Reading package lists... Done                      
> E: The repository 'https://nvidia.github.io/libnvidia-container/ubuntu18.04/amd64  Release' does not have a Release file.
> N: Updating from such a repository can't be done securely, and is therefore disabled by default.
> N: See apt-secure(8) manpage for repository creation and user configuration details.
> E: The repository 'https://nvidia.github.io/nvidia-container-runtime/ubuntu18.04/amd64  Release' does not have a Release file.
> N: Updating from such a repository can't be done securely, and is therefore disabled by default.
> N: See apt-secure(8) manpage for repository creation and user configuration details.
> E: The repository 'https://nvidia.github.io/nvidia-docker/ubuntu18.04/amd64  Release' does not have a Release file.
> N: Updating from such a repository can't be done securely, and is therefore disabled by default.
> N: See apt-secure(8) manpage for repository creation and user configuration details.

System details-
`uname -r`

> 4.15.0-55-generic

`uname -m`

> x86_64

`cat /etc/lsb-release`

> DISTRIB_ID=Ubuntu
> DISTRIB_RELEASE=18.04
> DISTRIB_CODENAME=bionic
> DISTRIB_DESCRIPTION="Ubuntu 18.04.2 LTS"

`docker --version`

> Docker version 19.03.1, build 74b1e89

`nvidia-smi -a`

==============NVSMI LOG==============

Timestamp                           : Fri Aug 16 11:56:15 2019
Driver Version                      : 410.48

Attached GPUs                       : 2
GPU 00000000:02:00.0
    Product Name                    : Quadro P6000
    Product Brand                   : Quadro
    Display Mode                    : Disabled
    Display Active                  : Disabled
    Persistence Mode                : Disabled
    Accounting Mode                 : Disabled
    Accounting Mode Buffer Size     : 4000
    Driver Model
        Current                     : N/A
        Pending                     : N/A
    Serial Number                   : 0323816186527
    GPU UUID                        : GPU-681a933d-7497-dfb9-fa98-0cd776485def
    Minor Number                    : 0
    VBIOS Version                   : 86.02.1D.00.01
    MultiGPU Board                  : No
    Board ID                        : 0x200
    GPU Part Number                 : 900-5G611-1700-000
    Inforom Version
        Image Version               : G611.0500.00.02
        OEM Object                  : 1.1
        ECC Object                  : 4.1
        Power Management Object     : N/A
    GPU Operation Mode
        Current                     : N/A
        Pending                     : N/A
    GPU Virtualization Mode
        Virtualization mode         : None
    IBMNPU
        Relaxed Ordering Mode       : N/A
    PCI
        Bus                         : 0x02
        Device                      : 0x00
        Domain                      : 0x0000
        Device Id                   : 0x1B3010DE
        Bus Id                      : 00000000:02:00.0
        Sub System Id               : 0x11A010DE
        GPU Link Info
            PCIe Generation
                Max                 : 3
                Current             : 3
            Link Width
                Max                 : 16x
                Current             : 16x
        Bridge Chip
            Type                    : N/A
            Firmware                : N/A
        Replays since reset         : 0
        Tx Throughput               : 0 KB/s
        Rx Throughput               : 0 KB/s
    Fan Speed                       : 21 %
    Performance State               : P0
    Clocks Throttle Reasons
        Idle                        : Not Active
        Applications Clocks Setting : Not Active
        SW Power Cap                : Not Active
        HW Slowdown                 : Not Active
            HW Thermal Slowdown     : Not Active
            HW Power Brake Slowdown : Not Active
        Sync Boost                  : Not Active
        SW Thermal Slowdown         : Not Active
        Display Clock Setting       : Not Active
    FB Memory Usage
        Total                       : 24449 MiB
        Used                        : 0 MiB
        Free                        : 24449 MiB
    BAR1 Memory Usage
        Total                       : 256 MiB
        Used                        : 2 MiB
        Free                        : 254 MiB
    Compute Mode                    : Default
    Utilization
        Gpu                         : 0 %
        Memory                      : 0 %
        Encoder                     : 0 %
        Decoder                     : 0 %
    Encoder Stats
        Active Sessions             : 0
        Average FPS                 : 0
        Average Latency             : 0
    FBC Stats
        Active Sessions             : 0
        Average FPS                 : 0
        Average Latency             : 0
    Ecc Mode
        Current                     : Disabled
        Pending                     : Disabled
    ECC Errors
        Volatile
            Single Bit            
                Device Memory       : N/A
                Register File       : N/A
                L1 Cache            : N/A
                L2 Cache            : N/A
                Texture Memory      : N/A
                Texture Shared      : N/A
                CBU                 : N/A
                Total               : N/A
            Double Bit            
                Device Memory       : N/A
                Register File       : N/A
                L1 Cache            : N/A
                L2 Cache            : N/A
                Texture Memory      : N/A
                Texture Shared      : N/A
                CBU                 : N/A
                Total               : N/A
        Aggregate
            Single Bit            
                Device Memory       : N/A
                Register File       : N/A
                L1 Cache            : N/A
                L2 Cache            : N/A
                Texture Memory      : N/A
                Texture Shared      : N/A
                CBU                 : N/A
                Total               : N/A
            Double Bit            
                Device Memory       : N/A
                Register File       : N/A
                L1 Cache            : N/A
                L2 Cache            : N/A
                Texture Memory      : N/A
                Texture Shared      : N/A
                CBU                 : N/A
                Total               : N/A
    Retired Pages
        Single Bit ECC              : N/A
        Double Bit ECC              : N/A
        Pending                     : N/A
    Temperature
        GPU Current Temp            : 44 C
        GPU Shutdown Temp           : 100 C
        GPU Slowdown Temp           : 97 C
        GPU Max Operating Temp      : N/A
        Memory Current Temp         : N/A
        Memory Max Operating Temp   : N/A
    Power Readings
        Power Management            : Supported
        Power Draw                  : 62.22 W
        Power Limit                 : 250.00 W
        Default Power Limit         : 250.00 W
        Enforced Power Limit        : 250.00 W
        Min Power Limit             : 125.00 W
        Max Power Limit             : 250.00 W
    Clocks
        Graphics                    : 1506 MHz
        SM                          : 1506 MHz
        Memory                      : 4513 MHz
        Video                       : 1354 MHz
    Applications Clocks
        Graphics                    : 1506 MHz
        Memory                      : 4513 MHz
    Default Applications Clocks
        Graphics                    : 1506 MHz
        Memory                      : 4513 MHz
    Max Clocks
        Graphics                    : 1657 MHz
        SM                          : 1657 MHz
        Memory                      : 4513 MHz
        Video                       : 1493 MHz
    Max Customer Boost Clocks
        Graphics                    : 1657 MHz
    Clock Policy
        Auto Boost                  : N/A
        Auto Boost Default          : N/A
    Processes                       : None

GPU 00000000:03:00.0
    Product Name                    : Quadro P6000
    Product Brand                   : Quadro
    Display Mode                    : Disabled
    Display Active                  : Disabled
    Persistence Mode                : Disabled
    Accounting Mode                 : Disabled
    Accounting Mode Buffer Size     : 4000
    Driver Model
        Current                     : N/A
        Pending                     : N/A
    Serial Number                   : 0323716050268
    GPU UUID                        : GPU-76b2493c-0e53-958c-b5f9-bf13d06c96e4
    Minor Number                    : 1
    VBIOS Version                   : 86.02.1D.00.01
    MultiGPU Board                  : No
    Board ID                        : 0x300
    GPU Part Number                 : 900-5G611-1700-000
    Inforom Version
        Image Version               : G611.0500.00.02
        OEM Object                  : 1.1
        ECC Object                  : 4.1
        Power Management Object     : N/A
    GPU Operation Mode
        Current                     : N/A
        Pending                     : N/A
    GPU Virtualization Mode
        Virtualization mode         : None
    IBMNPU
        Relaxed Ordering Mode       : N/A
    PCI
        Bus                         : 0x03
        Device                      : 0x00
        Domain                      : 0x0000
        Device Id                   : 0x1B3010DE
        Bus Id                      : 00000000:03:00.0
        Sub System Id               : 0x11A010DE
        GPU Link Info
            PCIe Generation
                Max                 : 3
                Current             : 3
            Link Width
                Max                 : 16x
                Current             : 16x
        Bridge Chip
            Type                    : N/A
            Firmware                : N/A
        Replays since reset         : 0
        Tx Throughput               : 1000 KB/s
        Rx Throughput               : 1000 KB/s
    Fan Speed                       : 20 %
    Performance State               : P0
    Clocks Throttle Reasons
        Idle                        : Not Active
        Applications Clocks Setting : Not Active
        SW Power Cap                : Active
        HW Slowdown                 : Not Active
            HW Thermal Slowdown     : Not Active
            HW Power Brake Slowdown : Not Active
        Sync Boost                  : Not Active
        SW Thermal Slowdown         : Not Active
        Display Clock Setting       : Not Active
    FB Memory Usage
        Total                       : 24449 MiB
        Used                        : 0 MiB
        Free                        : 24449 MiB
    BAR1 Memory Usage
        Total                       : 256 MiB
        Used                        : 2 MiB
        Free                        : 254 MiB
    Compute Mode                    : Default
    Utilization
        Gpu                         : 2 %
        Memory                      : 0 %
        Encoder                     : 0 %
        Decoder                     : 0 %
    Encoder Stats
        Active Sessions             : 0
        Average FPS                 : 0
        Average Latency             : 0
    FBC Stats
        Active Sessions             : 0
        Average FPS                 : 0
        Average Latency             : 0
    Ecc Mode
        Current                     : Disabled
        Pending                     : Disabled
    ECC Errors
        Volatile
            Single Bit            
                Device Memory       : N/A
                Register File       : N/A
                L1 Cache            : N/A
                L2 Cache            : N/A
                Texture Memory      : N/A
                Texture Shared      : N/A
                CBU                 : N/A
                Total               : N/A
            Double Bit            
                Device Memory       : N/A
                Register File       : N/A
                L1 Cache            : N/A
                L2 Cache            : N/A
                Texture Memory      : N/A
                Texture Shared      : N/A
                CBU                 : N/A
                Total               : N/A
        Aggregate
            Single Bit            
                Device Memory       : N/A
                Register File       : N/A
                L1 Cache            : N/A
                L2 Cache            : N/A
                Texture Memory      : N/A
                Texture Shared      : N/A
                CBU                 : N/A
                Total               : N/A
            Double Bit            
                Device Memory       : N/A
                Register File       : N/A
                L1 Cache            : N/A
                L2 Cache            : N/A
                Texture Memory      : N/A
                Texture Shared      : N/A
                CBU                 : N/A
                Total               : N/A
    Retired Pages
        Single Bit ECC              : N/A
        Double Bit ECC              : N/A
        Pending                     : N/A
    Temperature
        GPU Current Temp            : 53 C
        GPU Shutdown Temp           : 100 C
        GPU Slowdown Temp           : 97 C
        GPU Max Operating Temp      : N/A
        Memory Current Temp         : N/A
        Memory Max Operating Temp   : N/A
    Power Readings
        Power Management            : Supported
        Power Draw                  : 56.27 W
        Power Limit                 : 250.00 W
        Default Power Limit         : 250.00 W
        Enforced Power Limit        : 250.00 W
        Min Power Limit             : 125.00 W
        Max Power Limit             : 250.00 W
    Clocks
        Graphics                    : 405 MHz
        SM                          : 405 MHz
        Memory                      : 4513 MHz
        Video                       : 658 MHz
    Applications Clocks
        Graphics                    : 1506 MHz
        Memory                      : 4513 MHz
    Default Applications Clocks
        Graphics                    : 1506 MHz
        Memory                      : 4513 MHz
    Max Clocks
        Graphics                    : 1657 MHz
        SM                          : 1657 MHz
        Memory                      : 4513 MHz
        Video                       : 1493 MHz
    Max Customer Boost Clocks
        Graphics                    : 1657 MHz
    Clock Policy
        Auto Boost                  : N/A
        Auto Boost Default          : N/A
    Processes                       : None

**Comment by anshumang** [comment]
@3XX0 let me know if I should reopen a new issue

**Comment by franva** [comment]
> @3XX0 let me know if I should reopen a new issue

have you fixed it? I am having the same issue.

**Comment by RenaudWasTaken** [comment]
```
curl -s -L https://nvidia.github.io/nvidia-docker/gpgkey | \
  sudo apt-key add -
```

**Comment by franva** [comment]
> 
> 
> ```
> curl -s -L https://nvidia.github.io/nvidia-docker/gpgkey | \
>   sudo apt-key add -
> ```

thanks man, I will give it a try :)

**Comment by wanxinzzz** [comment]
> ```
> curl -s -L https://nvidia.github.io/nvidia-docker/gpgkey | \
>   sudo apt-key add -
> ```

Thanks, This work for me
